package-lock.json

```diff
@@ -2,90 +2,36 @@
   "requires": true,
```

Comment:
Did you intend to update some dependencies here?

Reply:
I didn't, I just follow the setup.md